src/routes/docs/+page.svelte

```diff
@@ -4,4 +4,6 @@
 </script>
 
 <DocsMain />
-<Explore />
+<div class="w-full bg-navy">
```

Comment:
Moving the bg to outside so that it can go full width and keep the contents of explore constrained